biz.aQute.bndlib.tests/test/test/servicecapability/ServiceCapabilityTest.java

```diff
@@ -0,0 +1,98 @@
+package test.servicecapability;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
This is a mis-mash of JUnit 4 and JUnit 5 types. Please change to all JUnit 5.

Reply:
switched to AssertJ